templates/prometheus.go

```diff
@@ -46,6 +46,7 @@ var PrometheusTemplate = promv1.Prometheus{
 		ServiceMonitorSelector: &resourceSelector,
 		PodMonitorSelector:     &resourceSelector,
 		RuleSelector:           &resourceSelector,
+		ExternalLabels:         map[string]string{"app": "managed-ocs"},
```

Comment:
- I believe it's better to update this in reconcile rather than mentioning it here 🤔